src/psyclone/tests/psyad/adjoint_visitor_test.py

```diff
@@ -781,6 +793,129 @@ def test_loop_logger(fortran_reader, caplog):
             "contains no active variables" in caplog.text)
 
 
+# AdjointVisitor.ifblock_node()
+
+def test_ifblock_node_active_error(fortran_reader):
+    '''Test that the ifblock_node method raises the expected exception
+    if no active variables are specified.
+
+    '''
+    tl_psyir = fortran_reader.psyir_from_source(TL_IF_CODE)
+    ifblock = tl_psyir.walk(IfBlock)[0]
```

Comment:
pylint points out that `ifblock` is unused.

Reply:
because of the error you point out in the next comment.